sefaria/model/schema.py

```diff
@@ -1479,10 +1479,15 @@ def first_child(self):
     def last_child(self):
         pass
 
+    def supports_language(self, lang):
```

Comment:
This is a strange case because `DictionaryEntryNode` and `SheetNode` don't inherit from `VirtualNode`, but share properties. I think we need a `support_language(lang)` function on all three, each with its own implementation.